### PR TITLE
WW-104 | Enabling pt filter on Special:PortabilityDashboard

### PR DESCRIPTION
**Pull request by rybmat**
Ticket: https://wikia-inc.atlassian.net/browse/WW-104

Per request from vanguard, enabling portugese (pt) language filter on Special:PortabilityDashboard.

/cc @Wikia/west-wing

### CODE REVIEW

**Comment by sebastianmarzjan** [comment]
Don't we want to adjust 'no results' message too?

**Comment by rybmat** [comment]
sure, forgot about that ;)

**Comment by rybmat** [comment]
added "no results" message, but if someone has better proposal for the copy, please share it

File: extensions/wikia/PortabilityDashboard/PortabilityDashboard.i18n.php [hunk 1]
```diff
@@ -54,5 +55,6 @@
 	'portability-dashboard-language-filter-all' => 'Language filter link - show all languages',
 	'portability-dashboard-special-insights-template-without-title' => 'Link title for unorganized templates on Special:Insights',
 	'portability-dashboard-special-insights-custom-infobox-title' => 'Link title for non-portable infoboxes on Special: Insights',
-	'portability-dashboard-refresh-frequency-info' => 'Portability data, content page views and migration impact are recalculated every Saturday at 9:00 GMT.'
+	'portability-dashboard-refresh-frequency-info' => 'Portability data, content page views and migration impact are recalculated every Saturday at 9:00 GMT.',
+	'portability-dashboard-no-results-info' => 'No results have been found for selected language.'
```
Comment (lukaszkonieczny) [inline extensions/wikia/PortabilityDashboard/PortabilityDashboard.i18n.php]: Can we make these qqq`s more descriptive ? ;)

File: extensions/wikia/PortabilityDashboard/PortabilityDashboard.i18n.php [hunk 2]
```diff
@@ -29,7 +29,8 @@
 	'portability-dashboard-language-filter-all' => 'all',
 	'portability-dashboard-special-insights-template-without-title' => 'Unorganized templates on Special:Insights',
 	'portability-dashboard-special-insights-custom-infobox-title' => 'Non-portable infoboxes on Special:Insights',
-	'portability-dashboard-refresh-frequency-info' => 'Portability data, content page views and migration impact are recalculated every Saturday at 9:00 GMT.'
+	'portability-dashboard-refresh-frequency-info' => 'Portability data, content page views and migration impact are recalculated every Saturday at 9:00 GMT.',
+	'portability-dashboard-no-results-info' => 'No results have been found for selected language.'
```
Comment (sebastianmarzjan) [inline extensions/wikia/PortabilityDashboard/PortabilityDashboard.i18n.php]: I recommend to update to match the description in the tickets more:
"We don't have any wikis in selected language that are in top {param} most significant communities from the migration impact perspective."

**Comment by lukaszkonieczny** [comment]
LGTM

**Comment by idradm** [comment]
LGTM

**Pull request merged**